packages/kit/src/core/dev/index.js

```diff
@@ -200,43 +200,95 @@ class Watcher extends EventEmitter {
 
 		/** @type {import('types/internal').SSRManifest} */
 		this.manifest = {
-			assets: manifest_data.assets,
-			layout: manifest_data.layout,
-			error: manifest_data.error,
-			routes: manifest_data.routes.map((route) => {
-				if (route.type === 'page') {
+			appDir: this.config.kit.appDir,
+			assets: new Set(manifest_data.assets.map((asset) => asset.file)),
+			_: {
+				mime: get_mime_lookup(manifest_data),
+				entry: {
+					file: `/${SVELTE_KIT}/dev/runtime/internal/start.js`,
+					css: [],
+					js: []
+				},
+				nodes: manifest_data.components.map((id) => {
+					return async () => {
```

Comment:
I think of a manifest as just being static data, so wouldn't quite expect to see a function in the manifest. We can't leave this as `load_component`?

Reply:
it makes things considerably more complex if the manifest isn't self-contained. am open to alternative naming suggestions, haven't come up with anything thus far

Reply:
renaming might not be a bad idea just because there's also the Vite `manifest.json` and it's sometimes confused me in the past that there are two things referred to as the manifest. Maybe `metadata`?

Reply:
Definitely not `metadata` 😅 — this is the meat of the app, or at least a slice of that meat (when we're talking about route splitting rather than a full manifest). It's more like 'content' or something, but I don't think that really captures it.

Reply:
`inventory`? `catalog`? `catalogue`? :grin: